### PR TITLE
[Docs] Update to latest canary

### PR DESCRIPTION
**Pull request by lfades**
Also added a change to make sure we fallback properly to the defined tag.

### CODE REVIEW

File: lib/docs/page.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { TAG, FORCE_TAG } from './config';
 export async function getCurrentTag(tag) {
   if (tag) return tag;
   if (FORCE_TAG) return TAG;
-  return getLatestTag();
+  return (await getLatestTag()) ?? TAG;
```
Comment (Timer) [inline lib/docs/page.js:9]: What's this fallback for? Seems dangerous and would potentially fallback to an old page.

Comment (lfades) [reply lib/docs/page.js:9]: @Timer You made me remember why did I remove it before! this is build time stuff, we want it to fail of course if fetching the latest release didn't work!

**Comment by zeit-team** [comment]
#### 📝Changed routes:


| [**/docs/\[...slug\]**](https://next-site-git-docs-v945-canary6.vercel.now.sh/docs/[...slug]) |
|:-:|
|<a href="https://next-site-git-docs-v945-canary6.vercel.now.sh/docs/[...slug]">Screenshot of /docs/[...slug]</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/Cn82X0xreXXiNv9lA614PYUNp1c_.png">(view full size)</a> |







Commit 0c9fc954359550cb29e15dd00eef0382ac7e8b1c (https://next-site-git-docs-v945-canary6.vercel.now.sh).

File: pages/docs/[...slug].js [hunk 2]
```diff
@@ -86,6 +86,7 @@ const Docs = ({ routes, route: _route, data, html }) => {
         router.push(`/docs${to || `/old#${hash}`}`);
       }
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (lfades) [inline pages/docs/[...slug].js:89]: it wants to add `router` to the deps, but that's not required for `router.push` to work properly.

**Pull request merged**